types/chrome/index.d.ts

```diff
@@ -436,7 +436,7 @@ declare namespace chrome.bookmarks {
 declare namespace chrome.browserAction {
     interface BadgeBackgroundColorDetails {
         /** An array of four integers in the range [0,255] that make up the RGBA color of the badge. For example, opaque red is [255, 0, 0, 255]. Can also be a string with a CSS value, with opaque red being #FF0000 or #F00. */
-        color: any;
+        color: string | ColorArray;
```

Comment:
`string | [number, number, number, number]`

Reply:
`browserAction.ColorArray` itself seems to be standardized as a part of WebExtension API.
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/browserAction/ColorArray